src/soc/sifive/fu540/src/phy.rs

```diff
@@ -0,0 +1,1220 @@
+/*:u32 =Copyright (c) 2018 SiFive, Inc */
+/*:u32 =SPDX-License-Identifier: Apache-2.0 */
+/*:u32 =SPDX-License-Identifier: GPL-2.0-or-later */
+/*:u32 =See the file LICENSE for further information */
```

Comment:
":u32 =" -- I think you went a bit overboard with find-and-replace in here.

Reply:
ha ha I man I was tired when I did that :-)